packages/blockchain-wallet-v4-frontend/src/components/BuySell/Coinify/Create/AcceptTerms/template.js

```diff
@@ -0,0 +1,246 @@
+                      defaultMessage='Send Again'
+                    />
+                  </SquaredButton>
+                  : null
+              }
+            </VerifiedContainer>
+            {
+              create === 'verify_email'
+                ? <SentEmailText size='14px' weight={300}>
+                  <FormattedMessage
+                    id='coinifyexchangedata.create.createaccount.partner.checkyourinbox'
+                    defaultMessage='Check your inbox. We just sent you an email.'
+                  />
+                </SentEmailText>
+                : null
+            }
+            <IconContainer>
+              {emailVerified ? (
+                <Fragment>
+                  <Icon
+                    name='checkmark'
+                    color='success'
+                    size='16px'
+                    weight={600}
+                  />
+                  <Text color='success' size='14px' weight={600}>
+                    <FormattedMessage
+                      id='coinifyexchangedata.create.createaccount.partner.emailverified'
+                      defaultMessage='Email Verified'
+                    />
+                  </Text>
+                </Fragment>
+              ) : null}
+            </IconContainer>
+            <TermsContainer>
+              {
+                emailVerified
+                  ? <Fragment>
+                    <TermsText size='12px' weight={300}>
+                      <FormattedMessage
+                        id='coinifyexchangedata.create.createaccount.partner.byclicking'
+                        defaultMessage="By clicking continue, I agree to Coinify's {ToS} & {Privacy}"
+                        values={{ ToS: <Link href='https://www.coinify.com/legal' size='12px' weight={300} rel='noreferrer noopener' target='_blank'>Terms of Service</Link>, Privacy: <Link href='https://www.coinify.com/legal/policy' size='12px' weight={300} rel='noreferrer noopener' target='_blank'>Privacy Policy</Link> }}
+                      />
+                    </TermsText>
+                    <Button nature='primary' type='submit' disabled={invalid || busy || signupError}>
+                      {
+                        !busy ? (
+                          <Text color='white' uppercase>
+                            <FormattedMessage
+                              id='continue'
+                              defaultMessage='Continue'
+                            />
+                          </Text>
+                        ) : (
+                          <HeartbeatLoader height='20px' width='20px' color='white' />
+                        )
+                      }
+                    </Button>
+                  </Fragment>
+                  : null
+              }
+            </TermsContainer>
+            <ErrorWrapper>
+              {signupError && (prop('error', signupError) === EMAIL_IN_USE_ERROR) && create !== 'verify_email'
```

Comment:
`signupError && (prop('error', signupError) === EMAIL_IN_USE_ERROR)` is same as just `prop('error', signupError) === EMAIL_IN_USE_ERROR`